src/pendulum/duration.py

```diff
@@ -273,9 +281,10 @@ def in_words(self, locale: str | None = None, separator: str = " ") -> str:
 
         if not parts:
             count: int | str = 0
-            if abs(self.microseconds) > 0:
-                unit = f"units.second.{loaded_locale.plural(1)}"
-                count = f"{abs(self.microseconds) / 1e6:.2f}"
+            unit = f"units.second.{loaded_locale.plural(0)}"
+            if self.microseconds != 0:
+                microseconds = abs(self.microseconds) / 1e6
+                count = f"{round(microseconds, ndigits=seconds_n_decimal)}"
```

Comment:
```suggestion
                count = str({round(microseconds, ndigits=seconds_n_decimal)})
```

or leave it as an int. Why converting it to string?

Reply:
'Cause I'm a dummy sometimes. Good call

Reply:
This is actually my very first time contributing to anything, probably should be int